api/test/run_test.sh

```diff
@@ -8,7 +8,7 @@ endcol="\e[0m"
 # ```
 # $	./test/run_test.sh test/audio_integration_test.py
 # ```
-TEST_ARG="${1:-test/}"
+if [ $# -ge 1 ]; then TEST_ARG="$@"; else TEST_ARG="test/"; fi
```

Comment:
Might as well split this across multiple lines :sweat_smile: 

```suggestion
if [ $# -ge 1 ]; then
    TEST_ARG="$@"
else
    TEST_ARG="test/"
fi
```